src/types/fixedtimezone.jl

```diff
@@ -95,3 +95,18 @@ end
 
 name(tz::FixedTimeZone) = tz.name
 rename(tz::FixedTimeZone, name::AbstractString) = FixedTimeZone(name, tz.offset)
+
+function Base.:(==)(a::FixedTimeZone, b::FixedTimeZone)
+    return a.name == b.name && a.offset == b.offset
+end
+
+function Base.isequal(a::FixedTimeZone, b::FixedTimeZone)
+    return isequal(a.name, b.name) && isequal(a.offset, b.offset)
+end
+
+function Base.hash(tz::FixedTimeZone, h::UInt)
+    h = hash(:timezone, h)
+    h = hash(tz.name, h)
+    h = hash(tz.offset, h)
+    return h
+end
```

Comment:
Why do we need to overload == and hash?

I expected making it isbits would decrease that need.